src/main/java/com/ibs/vehiclemanagementservice/model/VehicleDtls.java

```diff
@@ -0,0 +1,86 @@
+package com.ibs.vehiclemanagementservice.model;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+// Mongo database annotation.
+@Document(collection= "vehicledtls")
+public class VehicleDtls {
+
+	@Id
+	private int id;

```

Comment:
Is this the pass id?

Reply:
nope, this just an ID, We can make it as VehiclePass ID, Thats good idea. Admin while changing the status,  can generate unique no to this id.